engine/class_modules/sc_mage.cpp

```diff
@@ -3965,6 +3970,15 @@ struct arcane_missiles_t final : public custom_state_spell_t<arcane_mage_spell_t
     {
       p()->buffs.clearcasting_channel->trigger();
       p()->trigger_time_manipulation();
+      p()->buffs.intuition->trigger();
```

Comment:
Even though AM cannot be cast without CC, you shouldn't assume this will always be the case. Intuition wouldn't work properly then.